lib/DDG/Meta/Data.pm

```diff
@@ -122,6 +125,26 @@ sub get_ia {
     return $m;
 }
 
+sub _satisfy {
+	my ($ia, $by, $lookup) = @_;
+	return $by->($ia, $lookup) if ref $by eq 'CODE';
+	my $pby = $ia->{$by};
+	ref $lookup eq 'CODE' ? $lookup->($pby) : $pby eq $lookup;
+}
+
+# get_ias(repo => 'goodies', dev_milestone => 'live'...)
+# Lookups combine as an AND operation.
+sub get_ias {
+	my (@lookups) = @_;
+	my @results = @ia_container;
+	foreach (pairs @lookups) {
```

Comment:
It's standard really.  Speed/memory.  In this case it probably doesn't make a ton of difference since we're not expecting big data structures. You will need a reference for the value or you will have to check whether it's a ref or not.  

The `vs` should be a ref to a hash, i.e. `filter_ias({id => 'calculator'})` not an array.  Specifying a key twice would be an error.